acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -0,0 +1,31 @@
+step "Install PC1 repository" do
+  install_puppetlabs_release_repo(hosts, 'pc1')
+end
+
+step "Install legacy Puppet agents" do
+  default_puppet_version = '1.10.1'
+  puppet_version = ENV['PUPPET_LEGACY_VERSION']
+  if not puppet_version
+    logger.info "PUPPET_LEGACY_VERSION is not set!"
+    logger.info "  using default value of #{default_puppet_version}"
+    puppet_version = default_puppet_version
+  end
+
+  hosts.each { |host| host.install_package('puppet-agent', '', puppet_version) }
+end
+
+step "Install legacy Puppetserver" do
+  master.install_package('puppetserver', '', '2.7.2')
+end
+
+step "Configure puppet.conf" do
+  hostname = on(master, 'facter hostname').stdout.strip
+  fqdn = on(master, 'facter fqdn').stdout.strip
+  dir = master.tmpdir(File.basename('/tmp'))
+
+  lay_down_new_puppet_conf( master,
+                           {"main" => { "dns_alt_names" => "puppet,#{hostname},#{fqdn}",
+                                       "verbose" => true }}, dir)
+end
+
+puppetserver_initialize_ssl
```

Comment:
ah, n/m.  Looks like that already happens as part of the `puppetserver_initialize_ssl` helper [here](https://github.com/puppetlabs/puppetserver/blob/d47353d2ffde378f080844716af5cc7833a6eb2b/acceptance/lib/helper.rb#L133-L136).